index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio: Mobile First</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <header class="container-header flex-row">
+    <div class="logo"><img src="image/my-logo.png" alt="My Logo"></div>
+    <a href="https://github.com/joshield/setup-and-mobile-first"><nav class="menu flex-row"><img src="image/menu-btn.png" alt="Menu"></nav></a>
+  </header>
+
+  <section class="mid-section flex-column">
+
+    <h1>Hey There. <br> I'm Josh.</h1>
+
+    <h3>I'm a Software Developer</h3>
+
+    <p> I can help you build a product, feature or website Look through some of my work and experience!
+      If you like what you see and have a project you need coded, don't hesitate to contact me.
+    </p>
+
+    <section class="social">
+      <ul>
+      <li><a href="https://github.com/joshield" class="git"><img src="image/git.png" alt="git"></a></li>
+      <li><a href="https://www.linkedin.com/in/joshuaimmanuel/" class="linkedin"><img src="image/linkedin.png" alt="linkedin"></a></li>
+      <li><a href="https://medium.com/" class="vector"><img src="image/vector.png" alt=""></a></li>
+      <li><a href="https://twitter.com/joshield" class="twitter"><img src="image/twitter.png" alt="twitter"></a></li>
+      <li><a href="https://github.com/joshield" class="medium"><img src="image/medium.png" alt="git"></a></li>
+      </ul>
+    </section>
```

Comment:
- [ ] 🙏Kindly replace the `section` tag with `div` and ensure👊 that all sections have margins similar to the Figma design. This will help improve your design🎉 and make it look as cool as the Figma design.

Reply:
Done as requested.